RecoPixelVertexing/PixelTrackFitting/python/pixelFitterByBrokenLine_cfi.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this whole file is not needed if you set the default label to `"pixelFitterByBrokenLine"`